acceptance-tests/src/test/java/tech/pegasys/web3signer/tests/signing/SigningAcceptanceTestBase.java

```diff
@@ -38,28 +40,27 @@ protected void setupFilecoinSigner() {
     startSigner(builder.build());
   }
 
-  protected void setupEth2Signer() {
+  protected void setupEth2Signer(final Eth2Network eth2Network, final SpecMilestone specMilestone) {
     final SignerConfigurationBuilder builder = new SignerConfigurationBuilder();
     builder
         .withKeyStoreDirectory(testDirectory)
         .withMode("eth2")
-        .withAltairForkEpoch(MINIMAL_ALTAIR_FORK);
-    startSigner(builder.build());
-  }
+        .withNetwork(eth2Network.configName());
 
-  protected void setupEth2SignerMinimal() {
-    final SignerConfigurationBuilder builder = new SignerConfigurationBuilder();
-    builder
-        .withKeyStoreDirectory(testDirectory)
-        .withMode("eth2")
-        .withNetwork("minimal")
-        .withAltairForkEpoch(MINIMAL_ALTAIR_FORK);
-    startSigner(builder.build());
-  }
+    switch (specMilestone) {
+      case PHASE0:
+        break;
+      case ALTAIR:
+        builder.withAltairForkEpoch(MINIMAL_ALTAIR_FORK);
+        break;
+      case BELLATRIX:
+        builder.withAltairForkEpoch(MINIMAL_ALTAIR_FORK);
+        builder.withBellatrixForkEpoch(MINIMAL_ALTAIR_FORK);
```

Comment:
Shouldn't this should the bellatrix fork epoch here

Reply:
Added some comments and modified code slightly to make it more clear.